project/design.md

```diff
@@ -0,0 +1,25 @@
+This is a strawman proposal for the design of Merritt.
+
+# Goal
+The goal of `merritt/merritt` is to create an open source container management application.  This will be assembled from a variety of "extensions" that are implemented in containers.  Each service will provide a standard interface for implementation and a GRPC API for other services to use.
+
+# Components
```

Comment:
These are just the start.  We also need to add the ability to "mount" and support external components as part of the controller but this will give us a start.